src/Sarif.Driver/Sdk/AnalyzeOptionsBase.cs

```diff
@@ -111,7 +111,7 @@ public abstract class AnalyzeOptionsBase : CommonOptionsBase
         public string BaselineSarifFile { get; set; }
 
         [Option(
-            "postUri",
+            "post-uri",
             HelpText = "A URI to which the SARIF log file will be posted.")]
```

Comment:
>HelpText [](http://example.com/codeflow?start=12&length=8)

Where is my unit test? I will always ask a question if you propose a change without a unit test. In all gentleness, I'd like to report to you that I am growing tired of giving this note in pull requests. If you don't propose a test with your change, you must please explain why you don't believe it is necessary.

This is a well-tested area, I think, but clearly we don't test a top-down test that starts with the command-line. In general this is a weaker area of our testing.

Reply:
added 2 type of tests:
one is for command line, test change the string from postUri to post-uri;
the other is testing the function after command line is already parsed.